src/coffee/directives/api/models/child/window-child-model.coffee

```diff
@@ -30,10 +30,9 @@ angular.module('uiGmapgoogle-maps.directives.api.models.child')
           @scope.$on '$destroy', =>
             @destroy()
           $log.info @
-        #todo: watch model in here, and recreate / clean gObject on change
 
-        doShow: =>
-          if @scope.show == true
+        doShow: (wasOpen) =>
+          if @scope.show == true or wasOpen
```

Comment:
No not always. In the case of this example the Window is moving and its content needs to be updated. However sometimes `@scope.show` can be undefined. So in the case that show is not defined then wasOpen should trump it.